app/lnd/lib/lightning.js

```diff
@@ -16,7 +16,9 @@ process.env.GRPC_SSL_CIPHER_SUITES =
     'ECDHE-ECDSA-AES128-GCM-SHA256',
     'ECDHE-ECDSA-AES256-GCM-SHA384',
     'ECDHE-ECDSA-AES128-CBC-SHA256',
-    'ECDHE-ECDSA-CHACHA20-POLY1305'
+    'ECDHE-ECDSA-CHACHA20-POLY1305',
+    'ECDHE-RSA-AES256-GCM-SHA384',
+    'ECDHE-RSA-AES128-GCM-SHA256'
```

Comment:
Could you add a comment as to the need for these?

Reply:
I've added a comment to that effect.